### PR TITLE
Security doc

### PR DESCRIPTION
**Pull request by Pierozi**
this PR continue the initial #35 begin by Oswald

### CODE REVIEW

File: _docs/security/firewall.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+excerpt:        "Allowing continuousphp in your network"
+---
+
+The following IP addresses must be whitelisted in your firewall if you want to connect to continuousphp.
+This can be useful when ContinuousPHP Notification and Webhook try to reach your network.
+
+```
+52.44.91.48
+52.44.117.76
+52.44.72.35
```
Comment (fdewinne) [inline _docs/security/firewall.markdown]: a bullet list would be better here... This is not code

File: _docs/security/firewall.markdown [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+layout:         doc
+title:          "Firewall & Netowrk settings - Documentation"
+category:       "Firewall & Network settings"
+order:          20
+excerpt:        "Allowing continuousphp in your network"
+---
+
+The following IP addresses must be whitelisted in your firewall if you want to connect to continuousphp.
```
Comment (fdewinne) [inline _docs/security/firewall.markdown]: `to connect with` or `to authorize continuousphp accessing your network`

File: _docs/security/firewall.markdown [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+This can be useful when ContinuousPHP Notification and Webhook try to reach your network.
```
Comment (fdewinne) [inline _docs/security/firewall.markdown]: useful => helpful

File: _docs/security/firewall.markdown (hunk 2 above)
Comment (Pierozi) [inline _docs/security/firewall.markdown]: This word was already change by @ppaulis , the original was `You need to allow these IPs address if you like to allow continuousphp on your network.`

**Pull request merged**